### ISSUE TITLE
ALT gets "stuck" after ALT+TAB when Enable keyboard hook is active

### ISSUE THREAD

**Issue by hansek** [mzomparelli/zVirtualDesktop#295]
I'm on Windows 11 build 22000.376 and dealing with an issue when using ALT+TAB to switch applications/windows.

For example, I open two instances of Explorer window, use ALT+TAB to swich between them and as you I get to second Explorer window I have a focus on upper window Toolbar (copy, cut, paste, ...) but properly I should have the focus on window part with list of files/folders. 

Today I found out that disabling option `Enable keyboard hook` on `More settings` will resolve the issue and everything is working correctly.

**Comment by mzomparelli** [comment]
I'm waiting for my new PC to arrive that is compatible with Windows 11. I'll investigate this more then. The issue is probably not specific to Windows 11 though.

**Comment by SHHSSH** [comment]
> I'm waiting for my new PC to arrive that is compatible with Windows 11. I'll investigate this more then. The issue is probably not specific to Windows 11 though.

Correct, I also encounter this pretty regularly and have to lock screen temporarily. W10 Version 20H2 Build 19042.1466
